test/grid_tests.cc

```diff
@@ -37,7 +37,7 @@ TEST_CASE("split_box dissects boxes as expected - 3d", "[grid]") {
 
 template <int MergeDim, int Dims>
 void test_directional_merge(box_vector<Dims> unmerged, box_vector<Dims> merged) {
-	CAPTURE(MergeDim);
+		CAPTURE(MergeDim);
```

Comment:
?

Reply:
Our clang-format check is broken again. Fantastic!